server/views/pages/activePunishments.njk

```diff
@@ -0,0 +1,38 @@
+{% extends "./index.njk" %}
+{% from "../macros/conditionallyShow.njk" import conditionallyShow %}
+{% block body %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-three-quarters">
+            <h1>{{name}}’ active punishments</h1>
+        </div>
+    </div>
+
+    <dl class="govuk-summary-list">
```

Comment:
I don't think `<dl>` is the right element for this - seems like it would confuse accessibility tools as the content is meant to be a list of `<dt>` followed by `<dd>` tags - not sure this will know which `<dd>` belongs to which `<dt>`?
I think just a table would be better as it is tabular data.